scripts/facts-tracker/index.js

```diff
@@ -23,14 +23,27 @@ function exec(command, args) {
   return execFileSync(command, args, options).toString();
 }
 
-const isCI = !!process.env.TRAVIS_REPO_SLUG;
+const isCI = !!process.env.REPO_SLUG;
```

Comment:
think you need to either check REPO_SLUG and TRAVIS_REPO_SLUG, or update travis.yml to also set REPO_SLUG

Reply:
I don't think travis.yml exists anymore. TRAVIS_REPO_SLUG is still referenced since the .circleci/config file set TRAVIS_REPO_SLUG, which I renamed to REPO_SLUG to be more generic.